### ISSUE TITLE
in threads replies

### ISSUE THREAD

**Issue by bam241** [pshriwise/unitBot#13]
`unitbot` is right now replying in the channel when a value+unit is given in a thread.
it would be better to have `unitbot` replying within the thread when the measurement is given in the thread.

(I though about having all `unitbot` replies threaded, but it will destroy my teaching objective as nobody gone read the thread)

**Comment by pshriwise** [comment]
Yeah Kalin and I were messing around with that yesterday. I'm not sure how to get it to reply in threads, but I'll look into it!

**Comment by bam241** [comment]
thx
(I was not trying to push you, it was more to remember it...)

**Comment by pshriwise** [comment]
Waiting for CI to fail so I have some time right now, no worries :)

Sadly though I think that because threads are a relatively new feature in slack our slackbot API doesn't have the option to reply to a thread. I'll see if it's an issue on their repo. If it isn't, I'll bring it up.
